src/content/manifest/environment-variables.mdx

```diff
@@ -0,0 +1,104 @@
+---
+title: Environment variables
+description: Use Okteto's built-in environment variables to streamline your Okteto manifest.
+sidebar_label: Environment variables
+id: environment-variables
+---
+
+The Okteto manifest utilizes environment variables with each section supporting specific ENV VARs. Additional ENV VARs are added regularly and it's important to understand the existing ones and their usage. Currently, there are three types of ENV VARs to be aware of:
```

Comment:
There are two ways to use variables. One is in the manifest, they are expanded there. For example, you can do:
```
image: ${OKTETO_BUILD_API_IMAGE}
```
and okteto expands the right value.
You can also refer to these variables in your `deploy/destroy` commands. They are injected in the environment of these commands.
Finally, there are build-in variables, and user defined variables.